src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Validators/MergeBuildingRequestValidator.cs

```diff
@@ -0,0 +1,29 @@
+namespace BuildingRegistry.Api.BackOffice.Abstractions.Building.Validators
+{
+    using System.Linq;
+    using Be.Vlaanderen.Basisregisters.GrAr.Edit.Validators;
+    using FluentValidation;
+    using Requests;
+    using Validation;
+
+    public class MergeBuildingRequestValidator: AbstractValidator<MergeBuildingRequest>
+    {
+        public MergeBuildingRequestValidator()
+        {
+            RuleFor(x => x.GeometriePolygoon)
+                .Must(gml => GmlPolygonValidator.IsValid(gml, GmlHelpers.CreateGmlReader()))
+                .WithErrorCode(ValidationErrors.Common.InvalidBuildingPolygonGeometry.Code)
+                .WithMessage(ValidationErrors.Common.InvalidBuildingPolygonGeometry.Message);
+
+            RuleFor(x => x.SamenvoegenGebouwen)
```

Comment:
Do we need validation on the format of the PURI's given in `SamenvoegenGebouwen`